operator/config/samples/karmada.yaml

```diff
@@ -9,6 +9,7 @@ spec:
       local:
         imageRepository: registry.k8s.io/etcd
         imageTag: 3.5.3-0
+        replicas: 1
```

Comment:
Does the replicas is a required replicas? What's the default value when users do not set it?

Reply:
the replicas is required. we should be consistent with other component. if the field .spec is nil, we will be populated default value.  @Poor12

Reply:
ok, I find that default value is 1.